<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestaUm</class>
 <widget class="QMainWindow" name="RestaUm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RestaUm</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuModo">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionTradicional"/>
    <addaction name="actionCruz"/>
    <addaction name="actionMais"/>
    <addaction name="actionBanquinho"/>
    <addaction name="actionTriangulo"/>
    <addaction name="actionLosango"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAjuda"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuModo"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionTradicional">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tradicional</string>
   </property>
  </action>
  <action name="actionCruz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cruz</string>
   </property>
  </action>
  <action name="actionBanquinho">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Banquinho</string>
   </property>
  </action>
  <action name="actionTriangulo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triangulo</string>
   </property>
  </action>
  <action name="actionMais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mais</string>
   </property>
  </action>
  <action name="actionFlecha">
   <property name="text">
    <string>Flecha</string>
   </property>
  </action>
  <action name="actionLosango">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Losango</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
